zcash_client_sqlite/src/wallet.rs

```diff
@@ -675,6 +675,40 @@ pub(crate) fn get_sent_memo(
         .transpose()
 }
 
+/// Returns the minimum birthday height for accounts in the wallet.
+///
+/// TODO ORCHARD: we should consider whether we want to permit protocol-restricted accounts; if so,
+/// we would then want this method to take a protocol identifier to be able to learn the wallet's
+/// "Orchard birthday" which might be different from the overall wallet birthday.
```

Comment:
Don't expose the TODO in the public documentation:
```suggestion
//
// TODO ORCHARD: we should consider whether we want to permit protocol-restricted accounts; if so,
// we would then want this method to take a protocol identifier to be able to learn the wallet's
// "Orchard birthday" which might be different from the overall wallet birthday.
```